images/scripts/arch.install

```diff
@@ -59,6 +59,8 @@ if [ -n "$do_install" ]; then
     packages=$(find build-results -name "*.pkg.tar.zst")
     pacman -U --noconfirm $packages
 
+    firewall-cmd --add-service=cockpit --permanent
```

Comment:
FTR, it might be nicer to move the `systemctl enable --now firewalld` to the .install script, or drop it at all. Our other images also don't do this by default, and it's always nice to keep the pristine image as close to a default install as possible.